handler.js

```diff
@@ -1,12 +1,15 @@
-import { Coinbase, Sms } from './lib';
+import { Coinbase, Kraken, Sms } from './lib';
 
 const coinbase = new Coinbase();
+const kraken = new Kraken();
 const sms = new Sms();
 
 export const getExchangeRates = (event, context, callback) =>
   Promise.all([
     coinbase.getExchangeRates('Buy'),
     coinbase.getExchangeRates('Sell'),
+    kraken.getExchangeRates('Buy'),
+    kraken.getExchangeRates('Ask'),
```

Comment:
Would it be possible to make Kraken optional? Maybe through an ENVIRONMENT variable?

Maybe something like this?

```js
export const getExchangeRates = (event, context, callback) => {
  const promisesQueue = [
    coinbase.getExchangeRates('Buy'),
    coinbase.getExchangeRates('Sell'),
  ];

  if (process.env.KRAKEN_ENABLED === 'enabled') { // Default to disabled
    promisesQueue.push(kraken.getExchangeRates('Buy'));
    promisesQueue.push(kraken.getExchangeRates('Ask'));
  }

  return Promise.all(promisesQueue)
    .then(() =>
      callback(null, { statusCode: 200, body: { message: 'Collected exchange rates.' } }),
    ...
```